apps/dotcms-ui/src/app/api/services/dot-nav-logo/dot-nav-logo.service.spec.ts

```diff
@@ -28,4 +28,12 @@ describe('DotNavLogoService', () => {
             done();
         });
     });
+
+    it("should not set a logo if the logo string doesn't starts with /dA", (done) => {
```

Comment:
I would add another test with the prefix `/dA`

Reply:
@alfredo-dotcms Wouldn't it be redundant with this test? https://github.com/dotCMS/core-web/blob/e479330a878426f6b089f18aeb75c328ff3f277f/apps/dotcms-ui/src/app/api/services/dot-nav-logo/dot-nav-logo.service.spec.ts#L24-L30

Reply:
you're right it's already covered!